localstack-core/localstack/config.py

```diff
@@ -1038,6 +1038,15 @@ def populate_edge_configuration(
 
 # Whether the Next Gen APIGW invocation logic is enabled (handler chain)
 APIGW_NEXT_GEN_PROVIDER = os.environ.get("PROVIDER_OVERRIDE_APIGATEWAY", "") == "next_gen"
```

Comment:
nit: This wouldn't work if a user would use `PROVIDER_OVERRIDE_APIGATEWAYV2=next_gen`, right?
But I don't think this would be necessary, it's just a matter of communication with the users in our release notes / announcements.

Reply:
Exactly, yes, I think we will communicate only about this one override, to not confuse users, as this can get quite complicated quickly. The other idea would be to have a specific flag like `ENABLE_APIGW_NEXT_GEN` which would overwrite the 3 `PROVIDER_OVERRIDE_*`, not sure what is cleaner?

Reply:
No, I think it's totally fine to stick to how it is. This is just an intermediary solution, and no users will just jump on this without reading about this in the release notes or an announcement, where we need to make sure that it's clear with env var to set. 👍🏽